dashboard/src/components/DeploymentFormBody/DeploymentFormBody.tsx

```diff
@@ -288,6 +288,33 @@ class DeploymentFormBody extends React.Component<
     }
     this.setState({ restoreDefaultValuesModalIsOpen: false });
   };
+
+  private renderDiff = () => {
+    let oldValues = "";
+    let title = "";
+    let emptyDiffText = "";
+    if (this.props.deployedValues) {
+      // If there are already some deployed values (upgrade scenario)
+      // We compare the values from the old release and the new one
+      oldValues = this.props.deployedValues;
+      title = "New values to submit";
```

Comment:
"Difference from deployed version"? (matching how you've labelled the new deployment text below)